src/pytools/api/_api.py

```diff
@@ -567,7 +292,7 @@ def validate_type(
 def validate_element_types(
     iterable: T_Iterable,
     *,
-    expected_type: Union[Type, Tuple[Type, ...]],
+    expected_type: Union[type, Tuple[type, ...]],
```

Comment:
Why `Type` is changed to `type` here? Is it a convention? I see that mypy treats both the same way. 

E.g. this snippet is typechecks by mypy correctly:
```python
from typing import Type, Tuple

def test(a: Tuple[Type, type]):
    pass

if __name__ == '__main__':
    test((int,int))
```